harmony_netcdf_to_zarr/log_wrapper.py

```diff
@@ -0,0 +1,34 @@
+"""Simple wrapper to add a log to harmony."""
+
+from time import time
+
+from harmony.util import config
+from harmony.logging import build_logger
+
+c = config(validate=False)
+c = c._replace(app_name="netcdf-to-zarr")
+logger = build_logger(c, name='NetCDF-to-Zarr', stream=None)
+logger.propagate = False
+
+
+def get_logger():
+    """Return a logger to use in NetCDF-to-Zarr."""
+    return logger
```

Comment:
If you can show me how to use it as a decorator I'm all ears.  I couldn't see how to do that.

 Otherwise, I can yank it out the decorator functions and just throw in logging/timing statements into rechunk_zarr  or just get rid of the timing on that all together.  I only added back the decorator to mosaic_to_zarr because I was using it in rechunk. 
 
 It is kind of dumb to recreate all of that, I talked to Vinnie about a call to the harmony-service-lib that would give me the logger, but that hasn't materialized yet.

Reply:
Generally - the decorator feels like a nice clean way to do the logging. I'll maybe try and think about if there is an alternative to the lines above, but suspect I won't come up something without new functionality in `harmony-service-lib`.

Reply:
So I went with this 2941bba

Reply:
It's not a decorator anymore, but requires a lot less code and gives almost as much information.